Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: display popup when failing to connect to full lobby

### Description (*)
Added a feature to display a popup to the user when they fail to join a lobby because it is full.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2646](https://jira.unity3d.com/browse/MTT-2646)
### Manual testing scenarios
1. Modify LobbyCreationUI.cs line 36 to use a maxplayer count of 1 instead of 8.
2. Start to instances of Boss Room.
3. In one of them create a lobby.
4. In the other connect to the lobby using the connect key of the lobby.
5. Popup should appear telling you that the lobby is full.

### Contribution checklist
